app/javascript/src/components/Dashboard/Contacts/ContactMenuBar/index.jsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+
+import { Typography } from "neetoui/v2";
+import { MenuBar } from "neetoui/v2/layouts";
+import PropTypes from "prop-types";
```

Comment:
Defining `prop-types` makes sense for reusable components like `Card` or `neeto-ui` components. 
However, defining `prop-types` for a component that is not being reused but is being used at a single place has some trade-off. 
1. The task of maintaining `prop-types`. In general, a lot of people would be working in a project, and they would be updating code on the go. Maintaining `prop-types` becomes difficult in the long run for one-time use components. 
2. If we are defining `prop-types` at one place, then we need to make sure that we are defining it throughout to nix inconsistencies.